lib/GADS/Schema/Result/Report.pm

```diff
@@ -377,15 +430,25 @@ sub fields_for_render {
             name       => $_->name,
             is_checked => $checked{ $_->id },
         }
-    } $layout->all( user_can_read => 1 );
+    } $layout->all(user_can_read => 1);
 
     return \@fields;
 }
 
-sub _read_security_marking {
-    my $self = shift;
+sub _read_security_marking
+{   my $self = shift;
 
     return $self->security_marking || $self->instance->read_security_marking;
 }
 
+sub group_ids
```

Comment:
Only used within edit and add - otherwise, the search is within `Layout.pm` for group permissions for whom can view